src/test/java/com/google/sps/servlets/UserDataServletTest.java

```diff
@@ -172,13 +169,13 @@ public void testPostMethodRedirect() throws Exception {
     servletUnderTest.doPostWrapper(datastore, mockRequest, mockResponse);
 
     // Check that the mockResponse redirected property
-    verify(mockRequest, atLeast(1)).getParameter(USER_ID_PROPERTY);
     verify(mockResponse).sendRedirect("/profile.html?id=" + TEST_USER_ID);
```

Comment:
Ok, sorry, remind me why we expect it to redirect?

Reply:
The doPost method always redirects to the profile page, because user information is always being updated from either the Login page (in which case they should go to the profile page) or the profile page itself (in which case they should also go to the profile page).